docs/developer-guide.md

```diff
@@ -0,0 +1,23 @@
+## Dev Workflow
+* Checkout the repo (this assumes the basic GitHub setup already completed)
```

Comment:
I think the expectation is that people will fork the repo and make PRs from their own repo and not work off of this one.

Reply:
updated to suggest forking instead